### ISSUE TITLE
Strato in-flight freeze after 15min flight time (FW.: 248)

### ISSUE THREAD

**Issue by Altgenug1** [fhorinek/BB#241]
On my flight today, my Strato froze after almost 16 minutes. It couldn't be switched off with the off button either, I could only switch it off after the flight with cursor right + left + off. I couldn't find a crash_report.zip file on the Strato. Attached is the debug log. FW 248, HW 22, serial no.: 062A682D. Have just seen that there is the FW261 and will install it
[debug.log](https://github.com/fhorinek/BB/files/8335828/debug.log)
.

**Comment by kolupaev** [comment]
Reporting one more crash, on FW 261. 

Happened mid flight, approx around 10-20 minute. 
[crash_report.zip](https://github.com/fhorinek/BB/files/8405533/crash_report.zip)

**Comment by jonyii** [comment]
I got few of these freezes on Fw 261 recently.
Total 4 freezes in 2hours today. One required hard restart (this ticket), another 3 vario froze at some pitch and dials stopped updating (but it was possible to control strato and switch it off / on). Maybe related?

These hard  buttons are not exactly easy to press when its freezing and I'm wearing thick gloves, especially not the 3 button reset combo.

**Comment by kolupaev** [comment]
@fhorinek Sadly, that makes Strato unusable on longer flights. 

Any more data we can gather to make fixing it easier?

**Comment by fhorinek** [comment]
This issue caused by fanet list overflow is fixed in 262. If you encourage crash in 262 please provide the crash_report

**Comment by kolupaev** [comment]
Yay, thanks for fixing this one :) 

Here is one I've got on 262: 
Firmware: R0000262
Hardware revison: 22
[crash_report.zip](https://github.com/fhorinek/BB/files/8636291/crash_report.zip)

**Comment by fhorinek** [comment]
Thank you for the report, will look at it Monday 

notes for the crash:
```
tile_create->map_draw->list_free

lon1 = -1221156155
lon2 = N/A
lat1 = 476947955
lat2 = 476128955
step_x = 6006
step_y = 4095
zoom = 5
```

**Comment by kolupaev** [comment]
Indeed, I did have map widget in the background, although not on a screen I've used. 

I'll try disabling it, see if it changes something.

**Comment by fhorinek** [comment]
map tiles are generating regardless of the widgets on screen, I suggest to temporary renaming map folder to map_old
